.github/workflows/release.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: Install cargo-about
         shell: bash
         env:
-          CARGO_ABOUT_VERSION: 0.8.0
+          CARGO_ABOUT_VERSION: 0.8.2
           SHA256: "b53515a1c85f37810ae03ed03a984cde60282eb1fba08ad0888bd9a62f7d5629"
```

Comment:
This is probably going to not work because the SHA256 should be updated. I think the correct value is
```
797bee53a904aab73b31c923da521b72ed5d4ac950903c928f6be3ed236de8b2
```
but should double check from here: https://github.com/EmbarkStudios/cargo-about/releases/tag/0.8.2